trace_decoder/src/lib.rs

```diff
@@ -41,8 +41,6 @@
 #![warn(missing_debug_implementations)]
 #![warn(missing_docs)]
 
-zk_evm_common::check_chain_features!();
-
```

Comment:
Should be brought back as well for consistency.

Reply:
I think this isn't the right place for it - only one crate should check for these features, and `trace_decoder` doesn't even use these features (it only passes it through to `evm_arith`, which does the checking itself).

As an analogy:
```rust
use anyhow::ensure;

fn outer(left: u8, right: u8) -> anyhow::Result<u8> {
    ensure!(left == right);
    middle(left, right)
}
fn middle(left: u8, right: u8) -> anyhow::Result<u8> {
    ensure!(left == right);
    inner(left, right)
}
fn inner(left: u8, right: u8) -> anyhow::Result<u8> {
    ensure!(left == right);
    Ok(left + right)
}
```
I think that `outer` and `middle` are needlessly duplicating a check in `inner` - it's inner's contract, and so inner's job to check.

Reply:
> and trace_decoder doesn't even use these features

It will very soon though, but I guess we can indeed limit the features check if wanted